docs/paper/paper.md

```diff
@@ -0,0 +1,88 @@
+---
+title: TorchMetrics - Measuring Reproducibility in PyTorch
+tags:
+  - python
+  - deep learning
+  - pytorch
+authors:
+  - Nicki Skafte Detlefsen
+    affiliation: "1,2"
+  - Jirka Borovec
+    affiliation: "1"
+  - Justus Schock
+    affiliation: "1"
+  - Ananya Harsh Jha
+    affiliation: "1"
+  - Teddy Koker
+    affiliation: "1"
+  - Lucadiliello
```

Comment:
@lucadiliello ^^

Reply:
My name is "Luca Di Liello" and my affiliation "University of Trento". Thank you a lot for letting us be part of this paper :)

Reply:
mind rereview and eventually approve? :]

Reply:
I think I can not.

Reply:
I think you can, just it wont be green :heavy_check_mark:  but gray :white_check_mark: 
![image](https://user-images.githubusercontent.com/6035284/145366411-3fb7fd69-5c47-48db-951b-32f179638b3e.png)

Reply:
Ohhh didn't know about that option, done!